README.md

```diff
@@ -6,16 +6,78 @@ You can see which column families mostly affects disk utilization in near real t
 ![Screenshot](http://i.imgur.com/6rJm3TM.png)
 
 ## Usage
+- When using a properties file
+`java -jar ctop.jar -p [<propertyFilePath> when not passed ctop.properties will be loaded from classpath with argument -p]`
+- When not using a properties file
```

Comment:
Please change the order. Example without properties should be first in the list.